src/utils/searchResults.js

```diff
@@ -0,0 +1,19 @@
+/* @flow */
```

Comment:
I think this would be cleaner if this isn't a function. Having the function seems unnecessary. For example, instead of:
```
return searchResults(
  <Component>,
  shownUsers.length === 0,
  'No users found'
)
```
something like:
```
return shownUsers.length === 0 ? <EmptyState > : <Component>
```